### ISSUE TITLE
Rename PPSSPPSDL to PPSSPP

### ISSUE THREAD

**Issue by aaronjheng** [hrydgard/ppsspp#17362]
### What should happen

As the title.

### Who would this benefit

No one really.

### Platform (if relevant)

macOS

### Games this would be useful in

None

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by hrydgard** [comment]
Yeah I considered doing this for 1.15, but that would also rename it for Linux, and who knows how many people have setups that assume the binary name is PPSSPPSDL. 

Though I might still change it in the future, with some CMake logic could indeed make the change only for macOS.

**Comment by TotalCaesar659** [comment]
@hrydgard I think it's better to rename PPSSPP and PPSSPP-SDL to PPSSPP-QT and PPSSPP respectively, making second version prioritized (especially for Linux). The main reason is that Qt version has many long standing bugs that are not fixed yet (e.g. https://github.com/hrydgard/ppsspp/issues/10962 and https://github.com/hrydgard/ppsspp/issues/6803) but SDL version doesn't. So proposing second one may prevent bug reports from newcomers that launch just "PPSSPP" and don't know about differences.

**Comment by hrydgard** [comment]
Yeah that's a valid point. Nobody has showed up willing to properly maintain the Qt code, so it's been slowly rotting mostly, and rightly should be deprioritized.

I'll probably make the name change sometime after the 1.15 release process is done.

**Comment by TotalCaesar659** [comment]
@hrydgard Are there any plans to make these changes in 1.19.0 or later? SDL 3.2.0 is [released](https://github.com/libsdl-org/SDL/releases/tag/release-3.2.0) so if you plan to move it could be a great opportunity for this.

**Comment by hrydgard** [comment]
Yeah, kinda forgot about it. I think I'll make the change soon.
